packages/rlp/src/index.ts

```diff
@@ -1,16 +1,12 @@
-export type Input = string | number | bigint | Uint8Array | List | null | undefined
+export type Input = string | number | bigint | Uint8Array | Array<Input> | null | undefined
 
-// Use interface extension instead of type alias to
-// make circular declaration possible.
-export interface List extends Array<Input> {}
```

Comment:
Not sure if this removal is worth the risk of breaking something on this release, in case someone has imported and used? Maybe we just keep and remove along v3?

Reply:
Well, v3 is a breaking release so the types are different and not backward compatible